### ISSUE TITLE
"Target not set" Error when dragging and dropping resources in the tree

### ISSUE THREAD

**Issue by randomdave** [modxcms/revolution#13122]
### Summary

"Target not set" Error when dragging and dropping resources in the tree. Error box appears but no reports in the error log.

This issue occurred in MODX 2.3.4 and not fixed in 2.3.5 or 2.4.2 or 2.5.0 or 2.5.1. I installed these sequentially.
### Environment

MODX version 2.3.4.

Also reported on Forum here:
[https://forums.modx.com/thread/97674/error-target-not-set#dis-post-544080](url)
and here:
[https://forums.modx.com/thread/?thread=98164&page=3](url)
and here:
https://forums.modx.com/thread/97558/2-3-5-drag-n-drop-in-resource-tree

**Comment by digitalpenguin** [comment]
I can't replicate this. I'm not getting any error boxes. What are you dropping the resources on to get the error?

**Comment by sottwell** [comment]
Compare your upgraded site_content table to a new installation. Sometimes some upgrade steps miss some of the updated database fields.

**Comment by randomdave** [comment]
I think I have found the solution.

I've experienced a number of issues since upgrading and I decided to delete the manager folder and upload a fresh one from the repo and re-run setup. This appears to have fixed this issue and another one on my site.

I checked my tables against Bob's recommendations and all the ones he mentioned are present.
[http://bobsguides.com/blog.html/2016/06/27/missing-db-fields-(unknown-column)/](url)

I've not been able to check the site_content tables against a fresh install, but will try as a double check as soon as I can.
